### ISSUE TITLE
Sourcemaps points to wrong line 

### ISSUE THREAD

**Issue by jeroencornelissen** [stealjs/steal-tools#832]
Our generated sourcemaps seem wrong.
If an error occurs and you click on it in the console. It will always point to the last line of "sourcemapped" file.

Here is some clarification:

This is my `main.js` throwing an error at line 3 (foo is undefined):
```
'use strict';

const a = window.foo.length;

for(let i = 0; i < 10; i++) {
	console.log(i);
}
```

If I build it with `minify: false` and `sourceMaps: false` the error in the console leads me to the right position.

![01 minify-false-sourcemaps-false](https://user-images.githubusercontent.com/1894257/30178720-43a23008-940a-11e7-96c7-96ebc5ca76ce.png)

If I build with `sourceMaps: true` (and `minify: false`)
The error message says it fails at line 5.

![error-message](https://user-images.githubusercontent.com/1894257/30178754-5ff8c410-940a-11e7-8107-5c3d88d0f5f0.png)

But at line 5 (**sourcemapped main.js**) is the `for` loop and not the error.

![02 minify-false-sourcemaps-true](https://user-images.githubusercontent.com/1894257/30178802-86bff398-940a-11e7-9fc4-ce985ff62bc0.png)

**Comment by m-mujica** [comment]
@jeroencornelissen what varsion of steal-tools were you using?

I pushed your example code here https://github.com/m-mujica/steal-sourcemaps-issue

but I'm unable to replicate the issue you're reporting, it seems to work fine in Chrome 60.0.3112.113.

**Comment by jeroencornelissen** [comment]
Latest version. But I think before 1.5 it worked.

**Comment by pYr0x** [comment]
@m-mujica he said 
> If I build with sourceMaps: true (and --->>> minify: false <<<--- )

just wondering if this is the difference?

**Comment by m-mujica** [comment]
I tried with minify set to false and seemed to work fine.

**Comment by jeroencornelissen** [comment]
Hmm, I'll try tot investigate further.

**Comment by m-mujica** [comment]
![google chrome](https://user-images.githubusercontent.com/724877/30541626-9dcb91b4-9c52-11e7-86c9-31208b393054.png)

**Comment by jeroencornelissen** [comment]
@m-mujica I've created a PR on your branch with the issue.
https://github.com/m-mujica/steal-sourcemaps-issue/pull/1

When I add an `import` statement in `main.js` I have the issue described in first post.

**Comment by m-mujica** [comment]
@jeroencornelissen thank you! I'll take a look :)

**Comment by matthewp** [comment]
@m-mujica I had to make a fork of the source-map repo way back to take care of a bug that might not exist any more... I would try switching back to the source-map package and see if that does fix this.

**Comment by jeroencornelissen** [comment]
Any updates on this?

**Comment by m-mujica** [comment]
@jeroencornelissen I did spend some time on this but couldn't find a fix. Might be an issue with the babel -> escodegen transform.

I plan to continue working on this, maybe next week.

**Comment by m-mujica** [comment]
@jeroencornelissen I made some progress with the sourcemaps from ES6 modules.

It's still not ready to be released but I wonder if you have some time to test out the steal-tools branch I'm working on:

In your package.json, point steal-tools to my branch:

```json
{
  "steal-tools": "stealjs/steal-tools#sourcemaps" 
}
```

Re-install your dependencies and run the build. Please, let me know if that fixes it for you.

**Comment by jeroencornelissen** [comment]
@m-mujica No, stilll the same. Error points to line 8 (last line) instead of 5.

**Comment by m-mujica** [comment]
@jeroencornelissen weird, I wonder if you had to delete the node_modules and install again. 

I updated this repo https://github.com/m-mujica/steal-sourcemaps-issue, I just cloned it from scratch and tried in an incognito window.

![google chrome](https://user-images.githubusercontent.com/724877/30972404-ae67c4f2-a427-11e7-9dc7-589b4b746a6c.png)

**Comment by jeroencornelissen** [comment]
@m-mujica try setting `minify: true` in your `build.js`then it will fail.

**Comment by m-mujica** [comment]
@jeroencornelissen yeah, I was expecting that.... just wanted to confirm minify false was working...

**Comment by PieterSwitten** [comment]
Any updates on this issue? Having the same problem.
